<?xml version='1.0' encoding='utf-8'?>

<launch>
    <!--运行科大讯飞在线语音识别功能包-->
    <node pkg = "castlex_voice_system" type = "castlex_awake_node" name = "castlex_awake_node" output="screen">
        <param name="lgi_param" value="appid = 551d49a3, work_dir = ." />
        <param name="response" value="/home/castlex/.ws/castlex_voice_system/res/music/wozai.wav" />
        <param name="ssb_param_path" value="/home/castlex/.ws/castlex_voice_system/res/ivw/wakeupresource.jet" />
        <param name="audio_file_path" value="/home/castlex/.ws/castlex_voice_system/bin/audio/awake.wav" />
    </node>

</launch>
